player-tester/comparator.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```

Comment:
Cool, why not!
Though an infinite number of chart (like with a `+` button) would be best!